<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window_EditBook</class>
 <widget class="QDialog" name="Window_EditBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>261</width>
     <height>121</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>BookName</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Author</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Subject</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>New Column</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="2" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>321</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="0">
       <widget class="QPushButton" name="pushButton_EditBook">
        <property name="maximumSize">
         <size>
          <width>220</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 195, 55);</string>
        </property>
        <property name="text">
         <string>EditBook</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="lineEdit_BookName">
        <property name="maximumSize">
         <size>
          <width>220</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(85, 255, 184);</string>
        </property>
        <property name="placeholderText">
         <string>BookName</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="lineEdit_Subject">
        <property name="maximumSize">
         <size>
          <width>220</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(85, 255, 184);</string>
        </property>
        <property name="placeholderText">
         <string>Subject</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="lineEdit_AuthorName">
        <property name="maximumSize">
         <size>
          <width>220</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(85, 255, 184);
</string>
        </property>
        <property name="placeholderText">
         <string>Author</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>80</y>
     <width>317</width>
     <height>73</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLineEdit" name="lineEdit_BookId">
      <property name="maximumSize">
       <size>
        <width>220</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 85, 255);</string>
      </property>
      <property name="placeholderText">
       <string>Book ID</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_Search">
      <property name="maximumSize">
       <size>
        <width>110</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 195, 55);</string>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
      <property name="icon">
       <iconset resource="image.qrc">
        <normaloff>:/img/img/icons8-search-24 (1).png</normaloff>:/img/img/icons8-search-24 (1).png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
